src/packages/@ncigdc/components/Modals/UserProfileModal.js

```diff
@@ -0,0 +1,135 @@
+// @flow
+import React from 'react';
+import BaseModal from '@ncigdc/components/Modals/BaseModal';
+import { compose } from 'recompose';
+import { connect } from 'react-redux';
+import Highlight from 'react-highlighter';
+import { omit } from 'lodash';
+
+import Hidden from '@ncigdc/components/Hidden';
+import SearchIcon from 'react-icons/lib/fa/search';
+import { withTheme } from '@ncigdc/theme';
+import Check from '@ncigdc/theme/icons/Check';
+import { Row, Column } from '@ncigdc/uikit/Flex';
+import LocalPaginationTable from '@ncigdc/components/LocalPaginationTable';
+import EntityPageHorizontalTable from '@ncigdc/components/EntityPageHorizontalTable';
+import withRouter from '@ncigdc/utils/withRouter';
+
+const enhance = compose(
+  connect(({ auth: { user } }) => ({ user })),
+  withTheme,
+  withRouter,
+);
+
+const paginationPrefix = 'userProjectsTable';
+
+const UserProfileModal = ({
+  user: { username, projects },
+  theme,
+  push,
+  query,
+}) => {
+  const flattenedProjects = Object.keys(projects).reduce(
+    (acc, k) => ({ ...acc, ...projects[k] }),
+    {},
+  );
+  const allValues = [
+    ...new Set(
+      Object.keys(flattenedProjects).reduce(
+        (acc, projectId) => [...acc, ...flattenedProjects[projectId]],
+        [],
+      ),
+    ),
+  ];
+  const searchTerm = query[`${paginationPrefix}_search`] || '';
+
+  const data = Object.keys(flattenedProjects)
+    .filter(projectId =>
+      projectId.toUpperCase().includes(searchTerm.toUpperCase()),
+    )
+    .map((projectId, i) => ({
+      project_id: <Highlight search={searchTerm}>{projectId}</Highlight>,
+      ...allValues.reduce(
+        (acc, v) => ({
+          ...acc,
+          [v]: flattenedProjects[projectId].includes(v)
+            ? <span><Check /><Hidden>True</Hidden></span>
+            : <Hidden>False</Hidden>,
+        }),
+        {},
+      ),
+    }));
+  return (
+    <BaseModal title={`Username: ${username}`} closeText="Done">
+      <Column>
+        <Row style={{ justifyContent: 'center' }}>
+          <label htmlFor="filter-cases">
+            <div
+              style={{
+                borderTop: `1px solid ${theme.greyScale5}`,
+                borderLeft: `1px solid ${theme.greyScale5}`,
+                borderBottom: `1px solid ${theme.greyScale5}`,
+                borderRight: 0,
+                borderRadius: '4px 0 0 4px',
+                backgroundColor: `${theme.greyScale4}`,
+                width: '38px',
+                height: '100%',
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+            >
+              <SearchIcon size={14} />
+            </div>
+            <Hidden>filter cases</Hidden>
+          </label>
+          <input
```

Comment:
can you either pass in `searchTerm` as the initial value or reset the query param on open? 
right now if you search for something and reopen the modal the results are filters but the input is empty.